schemas/content/product-attribute.schema.json

```diff
@@ -0,0 +1,52 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/content/product-attribute",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Product Attribute",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Generic product attribute of a product in the product catalog.",
+  "definitions": {
+    "attribute": {
+      "properties": {
+        "xdm:key": {
```

Comment:
I agree with giving more context and guidance. Can an enumeration be open, since we can not foresee all possible attributes.

Reply:
See: #334

Reply:
Yeah, don't do this. This is the "bag of key-values" anti pattern spelled out. You can do better by just defining a `patternProperties` object with `://` (regex), so that any URL is a valid extension, but I second @cdegroot-adobe's point that we should standardize as much as we can.